config/example/admins.txt

```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
-# not_a_user - Admin
+amoryblaine - Host
```

Comment:
```suggestion
amoryblaine - Host
```

i think you forgot to comment this out for your local thing.

well, it's not your fault, I should've given you the proper config